docs/software/ml/pytorch.md

```diff
@@ -337,7 +337,7 @@ export OMP_NUM_THREADS=8 # (2)!
 # PyTorch environment variables #
 #################################
 export MASTER_ADDR=$(hostname) # (3)!
-export MASTER_PORT=6000
+export MASTER_PORT=29500
```

Comment:
How about adding a comment to this line about why?

Reply:
+1 @bcumming, wording? something like:
Please avoid common ports like 6000, 60000 or <1024 which may be used by another service and could cause a silent tcp socket timeout.
?